### PR TITLE
Fix the double execution of exec_always commands on config reloads

### PR DESCRIPTION
**Pull request by Emantor**
While at it, fix the reload message.

### CODE REVIEW

File: sway/config.c [hunk 1]
```diff
@@ -389,7 +389,7 @@ bool load_main_config(const char *file, bool is_active, bool validating) {
 	config_defaults(config);
 	config->validating = validating;
 	if (is_active) {
-		wlr_log(WLR_DEBUG, "Performing configuration file reload");
+		wlr_log(WLR_DEBUG, "Performing configuration file reload or validation");
```
Comment (emersion) [inline sway/config.c]: Can you use `validating` to display a more precise debug message?

Comment (Emantor) [reply sway/config.c]: Done.

**Comment by Emantor** [comment]
Now the message should be correct :+1:

**Comment by emersion** [comment]
Thanks!

**Pull request merged**